web/html/src/components/table/TableDataHandler.tsx

```diff
@@ -172,13 +178,16 @@ export class TableDataHandler extends React.Component<Props, State> {
     });
   }
 
-  updateData({ items, total }: PagedData) {
+  updateData({ items, total, selectedIds }: PagedData) {
     this.setState({ data: items, totalItems: total }, () => {
       const lastPage = this.getLastPage();
       if (this.state.currentPage > lastPage) {
         this.setState({ currentPage: lastPage });
       }
     });
+    if (this.props.onSelect) {
```

Comment:
This could also be `this.props.onSelect?.(selectedIds)`, but either way of writing is fine.

Reply:
oh! I'm not fluent enough in JS or TS to know all those fancy things. I'll fix it